docs/source/reference/frame.rst

```diff
@@ -243,4 +243,6 @@ specific plotting methods of the form ``DataFrame.plot.<kind>``.
    DataFrame.plot.line
    DataFrame.plot.pie
    DataFrame.plot.scatter
-   DataFrame.hist
+   DataFrame.plot.hist
```

Comment:
Seems like `DataFrame.plot.hist` is listed twice.
I'm not sure we should keep `DataFrame.hist`, though.

Reply:
@itholic can you fix this? `DataFrame.hist` seems in the pandas doc as an alias. Let's keep it for now.

Reply:
sure, fixed in #851 
Thanks for comment !!